includes/class-wc-payments-blocks-payment-method.php

```diff
@@ -74,20 +74,18 @@ public function get_payment_method_script_handles() {
 		WC_Payments::register_script_with_dependencies( 'WCPAY_BLOCKS_CHECKOUT', 'dist/blocks-checkout', [ 'stripe' ] );
 		wp_set_script_translations( 'WCPAY_BLOCKS_CHECKOUT', 'woocommerce-payments' );
 
-		if ( WC()->cart ) {
-			wp_add_inline_script(
-				'WCPAY_BLOCKS_CHECKOUT',
-				'var wcBlocksCheckoutData = ' . wp_json_encode(
-					[
-						'amount'         => WC()->cart->get_total( '' ),
-						'currency'       => get_woocommerce_currency(),
-						'storeCountry'   => WC()->countries->get_base_country(),
-						'billingCountry' => WC()->customer->get_billing_country(),
-					]
-				) . ';',
-				'before'
-			);
-		}
+		wp_add_inline_script(
+			'WCPAY_BLOCKS_CHECKOUT',
+			'var wcBlocksCheckoutData = ' . wp_json_encode(
+				[
+					'amount'         => WC()->cart ? WC()->cart->get_total( '' ) : 0,
+					'currency'       => get_woocommerce_currency(),
+					'storeCountry'   => WC()->countries->get_base_country(),
+					'billingCountry' => WC()->customer ? WC()->customer->get_billing_country() : 'US',
+				]
+			) . ';',
+			'before'
+		);
```

Comment:
It looks like the `wcBlocksCheckoutData` is now added to any editor page that contains a block. But I noticed that on `develop` we're also including some scripts where we technically shouldn't 😬 

It looks like you added some checks on `WC()->cart` and `WC()->customer`, which were my main concern 👍

Reply:
Yeah, it's something to look at for sure.